x-pack/metricbeat/module/googlecloud/stackdriver/metricset.go

```diff
@@ -46,26 +46,29 @@ type config struct {
 	Metrics             []string `config:"stackdriver.metrics" validate:"required"`
 	Zone                string   `config:"zone" validate:"required"`
 	ProjectID           string   `config:"project_id" validate:"required"`
-	IncludeLabels       bool     `config:"include_labels"`
+	ExcludeLabels       bool     `config:"exclude_labels"`
```

Comment:
I meant to set `IncludeLabels` to true by default :smile: But this also works for me.

Reply:
I was wondering why it's called `ExcludeLabels` instead of  `IncludeLabels ` 😂